src/commands/uplink/NewEvent.ts

```diff
@@ -21,7 +43,7 @@ interface IEventBatteryAlarm extends IEventBase {
 }
 
 interface IEventActivateMtx extends IEventTime {
-    mtxAddress: number
+    mtxAddress: string
```

Comment:
what is this parameter?

maybe it's just an int64?

should we accept here number|string|Uint8Array?

Reply:
it's unique identifier or address like mac address, but for MTX meters and only 4 bytes